src/AutoMapper/Configuration/Annotations/IgnoreAttribute.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace AutoMapper.Configuration.Annotations
+{
+    /// <summary>
+    /// Ignore this member for configuration validation and skip during mapping.
+    /// </summary>
+    /// <remarks>
+    /// Must be used in combination with <see cref="AutoMapAttribute" />
+    /// </remarks>
+    public sealed class IgnoreAttribute : Attribute, IMemberConfigurationProvider
```

Comment:
`[AttributeUsage(AttributeTargets.Field | AttributeTargets.Property)]` ? Wherever is applies.